chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -774,6 +774,7 @@ fileInterface.viewSupportedFormats = View Supported Formats…
 fileInterface.untitledBibliography = Untitled Bibliography
 fileInterface.bibliographyHTMLTitle = Bibliography
 fileInterface.importError 		= An error occurred while trying to import the selected file. Please ensure that the file is valid and try again.
+fileInterface.importError.translator = An error occurred while trying to import the selected file with “%S”. Please ensure that the file is valid and try again.
```

Comment:
We can be a little less verbose:

`An error occurred importing the selected file with “%S”. Please ensure that the file is valid and try again.`